app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,50 @@
+class CommentsController < ApplicationController
+  before_action :authenticate_current_user!
+  before_action :set_comment, only: %w[create edit update]
+  before_action -> { authorize @comment }, only: %i[show edit update]
+
+  def index
+  end
+
+  def create
+    authorize Comment, :create?
+    comment = Comment.new(comment_params)
+
+    if comment.save
+      redirect_to tasks_path, notice: 'Comment was successfully created.'
+    else
+      redirect_to comment.task, alert: 'Comment was not created'
```

Comment:
How does user know what exactly is wrong with the comment?

Reply:
Will be fixed